test/server/cards/05_LOF/units/GrapplingGuardian.spec.ts

```diff
@@ -0,0 +1,28 @@
+describe('Grappling Guardian', function () {
+    integration(function (contextRef) {
+        it('Grappling Guardian\'s ability should allow defeating a space unit with 6 or less remaining HP', async function () {
+            await contextRef.setupTestAsync({
+                phase: 'action',
+                player1: {
+                    hand: ['grappling-guardian'],
+                },
+                player2: {
+                    spaceArena: [{
+                        card: 'devastator#inescapable',
+                        damage: 4
+                    }, 'avenger#hunting-star-destroyer', 'green-squadron-awing'],
+                    groundArena: ['wampa']
+                }
+            });
+            const { context } = contextRef;
+            const { player1, player2 } = context;
+
+            player1.clickCard(context.grapplingGuardian);
+            expect(player1).toBeAbleToSelectExactly([context.devastator, context.greenSquadronAwing]);
```

Comment:
It'd be good to check that this is an optional prompt and that it can target friendly units too.